docs/src/SelectList.doc.js

```diff
@@ -24,18 +24,28 @@ card(
       },
       {
         name: 'errorMessage',
-        type: '?string',
```

Comment:
Removed these, since we don't do this anywhere else. We should use the `required` option instead